tests/test.cpp

```diff
@@ -2865,10 +2868,10 @@ void FlexBuffersTest() {
   flexbuffers::Builder slb(512,
                            flexbuffers::BUILDER_FLAG_SHARE_KEYS_AND_STRINGS);
 
-  // Write the equivalent of:
-  // { vec: [ -100, "Fred", 4.0, false ], bar: [ 1, 2, 3 ], bar3: [ 1, 2, 3 ],
-  // foo: 100, bool: true, mymap: { foo: "Fred" } }
-  // clang-format off
+// Write the equivalent of:
```

Comment:
I am not sure why it would put comments inside a function in column 1. Unmatched clang-format command?

Reply:
idk, but its not that

Reply:
I ran git-clang-format against master and they were unmoved